backend/fms_core/template_importer/importers/_generic.py

```diff
@@ -46,14 +57,30 @@ def import_template(self, file, format, dry_run):
                         reversion.set_comment("Template import - dry run")
                     transaction.set_rollback(True)
                 else:
-                    self.import_template_inner()
+                    # Save template on disk with modified name (append timestamp and id) and insert path in imported_file
+                    if user is not None:
+                        os.environ["TZ"] = settings.LOCAL_TZ
+                        time.tzset()
+                        new_file_name = f"{file_name}_{time.strftime('%Y-%m-%d_%H:%M:%S')}_{user.username}{file_format}"
```

Comment:
The `:` character is illegal on OS X and Windows, so a different separator is needed. 
FYI:
The following characters are invalid as file or folder names on Windows using NTFS:
/ ? < > \ : * | " and any character you can type with the Ctrl key

The only illegal character for file and folder names in Mac OS X is the colon ":"

The only illegal character for file and folder names in Linux is "/"

Reply:
My bad some dash should do it.